### PR TITLE
feat: required fields in ctype wrapper schema

### PR DESCRIPTION
**Pull request by tjwelde**
## fixes KILTProtocol/ticket#204
This PR gets the required fields from the ctype schema and adds it in the schema section of the wrapper schema. Before that, some fields were not marked as required.

## How to test:
- Tests should run
- Making a ctype and using it to make a claim in demo-client has to be possible
  - To test, cherry-pick this commit onto sdk master branch and use it in the demo-client

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: src/ctype/CTypeSchema.ts [hunk 1]
```diff
@@ -16,12 +16,11 @@ export const CTypeModel = {
...
       type: 'string',
       format: 'uri',
-      default: 'http://kilt-protocol.org/draft-01/ctype#',
-      enum: ['http://kilt-protocol.org/draft-01/ctype#'],
+      const: 'http://kilt-protocol.org/draft-01/ctype#',
     },
     type: {
       type: 'string',
-      enum: ['object'],
+      const: 'object',
```
Comment (Dudleyneedham) [inline src/ctype/CTypeSchema.ts:23]: Is this a new JSON schema keyword?

Comment (tjwelde) [reply src/ctype/CTypeSchema.ts:23]: It was already available in json draft-07, which we are still using. We just didn't use `const`.

Comment (Dudleyneedham) [reply src/ctype/CTypeSchema.ts:23]: Awesome!

Comment (Dudleyneedham) [reply src/ctype/CTypeSchema.ts:23]: We will need to change the demo-client accordingly. 

Do we need a ticket?

Comment (tjwelde) [reply src/ctype/CTypeSchema.ts:23]: I checked the demo-client and no change is needed.

**Pull request merged**